cfn-resources/cluster/test/cfn-test-create-inputs.sh

```diff
@@ -20,34 +20,45 @@ if [[ "$*" == help ]]; then usage; fi
 
 rm -rf inputs
 mkdir inputs
-projectName="${1}"
-projectId=$(mongocli iam projects create "${projectName}" --output=json | jq -r '.id')
-echo "Created project \"${projectName}\" with id: ${projectId}"
 
-clusterName="${projectName}"
+projectName="test-project-for-cfn"
```

Comment:
Project name is fetched from env variable, but here I could see its hardcoded to "test-project-for-cfn", Ideally its better to have project name fetched from cfn-testing-helper.sh

Reply:
fixed